crypto/x509/x509_cmp.c

```diff
@@ -83,36 +83,6 @@ int X509_issuer_and_serial_cmp(const X509 *a, const X509 *b)
     return (X509_NAME_cmp(ai->issuer, bi->issuer));
 }
 
-unsigned long X509_issuer_and_serial_hash(X509 *a)
```

Comment:
Okay, we decided to remove this?

Reply:
Not unused.
https://github.com/awslabs/aws-lc/pull/94/commits/ca2162d71902385dc740a385d20977d551d14e8f

Reply:
i don't think you can use a 3rd party statement about no usage to conclude there are no usage internally.

Reply:
I searched internal code base with the filter that excludes `x509_cmp.c`, `x509.h` and `ssleay.txt`.

Reply:
That's better validation.